e2e-tests/pipeline_validation.sh

```diff
@@ -210,18 +222,51 @@ function test_parquet_sink() {
   rowcount "${HOME_PATH}/${PARQUET_SUBDIR}/Observation/" | awk '{print $3}')
   print_message "Total obs synced to parquet ---> ${total_obs_streamed}"
 
-  if [[ "${total_patients_streamed}" == "${TOTAL_TEST_PATIENTS}" && "${total_encounters_streamed}" \
-        == "${TOTAL_TEST_ENCOUNTERS}" && "${total_obs_streamed}" == "${TOTAL_TEST_OBS}" ]] \
-    ; then
-    print_message "PARQUET SINK EXECUTED SUCCESSFULLY USING ${PARQUET_SUBDIR} MODE"
-  else
-    print_message "PARQUET SINK TEST FAILED USING ${PARQUET_SUBDIR} MODE"
-    exit 1
+  if [[ ! (-n ${STREAMING}) ]]; then
+    print_message "Parquet Sink Test Streaming ONLY"
```

Comment:
I assume this is non-streaming mode, if yes can you please update the logs.

Reply:
Done.